src/components/button/Button.js

```diff
@@ -38,7 +38,7 @@ Button.propTypes = {
   type: PropTypes.oneOf(['primary', 'secondary', 'tertiary', 'disabled', 'danger']),
   typography: PropTypes.oneOf(['uppercase', 'capitalize', 'lowercase', 'none']),
   shape: PropTypes.oneOf(['square', 'rounded', 'pill']),
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
Agreed, but there's a warning in the console because content is null before the translation are loaded :( that's why I removed the `required`. Are we ok with that warning there?

Reply:
hmmm. Should we display anything on the page of there is no `content`?